Fix these problems:

implicit declaration of function 'read' is invalid in C99
implicit declaration of function 'write' is invalid in C99
implicit declaration of function 'close' is invalid in C99
implicit declaration of function 'make_dashed' is invalid in C99
implicit declaration of function 'set_cursor_busy' is invalid in C99

https://sourceforge.net/p/xournal/bugs/213/
--- src/ttsubset/sft.c.orig	2012-06-30 15:26:04.000000000 -0500
+++ src/ttsubset/sft.c	2020-09-09 08:45:03.000000000 -0500
@@ -43,9 +43,16 @@
  * @version 1.0
  */
 
+#ifdef HAVE_CONFIG_H
+#  include <config.h>
+#endif
+
 #include <assert.h>
 #include <stdlib.h>
 #include <string.h>
+#ifdef HAVE_UNISTD_H
+#include <unistd.h>
+#endif
 #include <fcntl.h>
 #include "sft.h"
 #ifdef USE_GSUB
--- src/ttsubset/ttcr.c.orig	2012-06-30 15:26:04.000000000 -0500
+++ src/ttsubset/ttcr.c	2020-09-09 08:38:47.000000000 -0500
@@ -44,6 +44,10 @@
  *
  */
 
+#ifdef HAVE_CONFIG_H
+#  include <config.h>
+#endif
+
 #include <sys/types.h>
 #ifdef HAVE_UNISTD_H
 #include <unistd.h>
--- src/xo-image.c.orig	2015-08-10 23:15:10.000000000 -0500
+++ src/xo-image.c	2020-09-09 08:54:21.000000000 -0500
@@ -25,6 +25,8 @@
 #include "xo-support.h"
 #include "xo-image.h"
 #include "xo-misc.h"
+#include "xo-paint.h"
+#include "xo-selection.h"
 
 // create pixbuf from buffer, or return NULL on failure
 GdkPixbuf *pixbuf_from_buffer(const gchar *buf, gsize buflen)
